spec/webauthn/attestation_statement/android_safetynet_spec.rb

```diff
@@ -21,7 +21,8 @@
       )
     end
 
-    let(:payload) { { "nonce" => nonce, "ctsProfileMatch" => cts_profile_match } }
+    let(:payload) { { "nonce" => nonce, "ctsProfileMatch" => cts_profile_match, "timestampMs" => timestamp } }
```

Comment:
We can multiply timestamp by `1000` here and avoid multiple multiplications elsewhere, right?

Reply:
I like it!